elifetools/tests/basic_usage_test.py

```diff
@@ -1,10 +1,6 @@
 import os
 import unittest
-
-os.sys.path.insert(0, os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
no idea what this was doing but looks unnecessary :+1:

Reply:
seems an hack for using "relative" imports, good that now we use absolute imports like `import elifetools.parseJATS`